api_tests/conftest.py

```diff
@@ -5,23 +5,32 @@
 from tests.json_api_test_app import JSONAPITestApp
 
 # Silence some 3rd-party logging and some "loud" internal loggers
-SILENT_LOGGERS = [
-    'api.caching.tasks',
-    'factory.generate',
-    'factory.containers',
-    'framework.analytics',
-    'framework.auth.core',
-    'framework.celery_tasks.signals',
-    'website.app',
-    'website.archiver.tasks',
-    'website.mails',
-    'website.notifications.listeners',
-    'website.search.elastic_search',
-    'website.search_migration.migrate',
-    'website.util.paths',
-]
-for logger_name in SILENT_LOGGERS:
-    logging.getLogger(logger_name).setLevel(logging.CRITICAL)
+# SILENT_LOGGERS = [
+#     'api.caching.tasks',
+#     'factory.generate',
+#     'factory.containers',
+#     'framework.analytics',
+#     'framework.auth.core',
+#     'framework.celery_tasks.signals',
+#     'website.app',
+#     'website.archiver.tasks',
+#     'website.mails',
+#     'website.notifications.listeners',
+#     'website.search.elastic_search',
+#     'website.search_migration.migrate',
+#     'website.util.paths',
+#     'osf.migrations.0001_initial',
+#     'osf.models',
+#     'addons.osfstorage.models',
+#     'rest_framework.pagination',
+#     'tests.base',
+#     'website.project',
+#     'django.db.models.fields',
+# ]
+# for logger_name in SILENT_LOGGERS:
+#     logging.getLogger(logger_name).setLevel(logging.CRITICAL)
+
+logging.disable(logging.CRITICAL)
```

Comment:
From the docs:

>  Its effect is to disable all logging calls of severity lvl and below, so that if you call it with a value of INFO, then all INFO and DEBUG events would be discarded, whereas those of severity WARNING and above would be processed according to the logger’s effective level.

This means that we would be ignoring CRITICAL messages, which we don't want to do. I think `logging.WARNING` would makes sense here.